### ISSUE TITLE
feat(QPBT/Algebra): prove the qudit-to-qubit Pauli isometry

### ISSUE THREAD

**Issue by Dengnifer** [Dengnifer/MIPStarRE-A#69]
### Motivation

Complete `lem:pauli-binary`, the qudit-to-qubit identification used by QPBT
soundness. The source statement is
`references/qpbt-paper/04_preliminaries.tex:1163-1199`, with proof at lines
1201-1262. The matched blueprint node is
`blueprint/src/chapter/ch11_qpbt_algebra.tex:711-781`.

### Target declaration

Prove only `MIPStarRE.QPBT.exists_qubitIsometry` in
`MIPStarRE/QPBT/Algebra/PauliTheorems.lean`, preserving its public signature
exactly. Preserve `qubitPauliProj` and the existing `**Local fix:**` paragraph
correcting the source's erroneous factor range `j in {1, ..., q}` to
`Fin F.basisDim`, where `basisDim = log2 q`.

### Dependencies and serialization

- Base this packet on merged #64, which proves
  `pauliProj_eq_avg_tauObservable`.
- Base it on merged #66, which proves the self-dual trace and coordinate
  identities, especially `binTrace_mul_eq_dotProduct`.
- Start only after #68 has merged because both packets own
  `PauliTheorems.lean`. This is a file-ownership dependency, not a mathematical
  dependency on #68's Pauli-relation or cancellation targets.
- The theorem receives `F : FixedFieldModel q` as data and does not depend on
  #65's proof that such models exist.

### Proof route

1. Define a private label equivalence by applying
   `F.binaryCoordinates.toEquiv` at each `Fin L` coordinate and then
   uncurrying:

   ​`​`​`lean
   (Equiv.piCongrRight (fun _ : Fin L => F.binaryCoordinates.toEquiv)).trans
     (Equiv.curry (Fin L) (Fin F.basisDim) (ZMod 2)).symm
   ​`​`​`

   Lift it to `phi` with `LinearIsometryEquiv.piLpCongrLeft 2 C C`.

2. Prove a private generic EPR transport lemma for a finite equivalence. Expand
   `isometryTensor` and `eprState`, use
   `EuclideanSpace.piLpCongrLeft_single`, rewrite the normalization with
   `Fintype.card_congr`, and collapse the two permutation sums with
   `Fintype.sum_eq_single`.

3. Prove a private generic conjugation lemma identifying conjugation by the
   inverse coordinate isometry with `Matrix.reindex` by the inverse label
   equivalence. Collapse the two matrix-product sums explicitly. Keep the
   orientation: `phi` maps qudit labels to bit labels, while `phi.symm`
   realizes the paper's `phi^dagger M phi` on the qudit space.

4. Establish private phase compatibility for `a,u : Fin L -> F.K` by combining
   trace linearity, `binTrace_mul_eq_dotProduct F`,
   `Algebra.trace_self_apply`, and `Fintype.sum_prod_type`.

5. Prove entrywise transport of `tauObservable` for both `PauliKind.X` and
   `.Z`. Use linearity and injectivity of `F.binaryCoordinates` in the shift
   case and phase compatibility in the phase case.

6. Rewrite both projectors with #64's
   `pauliProj_eq_avg_tauObservable`, transport the finite sum along the label
   equivalence with `Fintype.sum_equiv`, rewrite its normalization by
   `Fintype.card_congr`, and apply observable and phase transport termwise.

The coordinate equivalence, its `piLpCongrLeft` lift, generic EPR transport,
and generic conjugation/reindex proof have been scratch-elaborated successfully
against the current imports.

### Statement integrity

The paper assumes an admissible binary field size `q = 2^k`, its fixed
self-dual normal basis, and a nonnegative tensor exponent. `FixedFieldModel q`
packages exactly that field/basis boundary, and `L : Nat` includes the paper's
empty tensor-power case. The two Lean conclusions are exactly the source EPR
and `X`/`Z` projector identities. Verdict: **faithful boundary hypotheses**,
with the already documented source index correction; no extra assumptions and
no weakened or strengthened conclusion.

### Scope and integrity constraints

- Own only the proof of `exists_qubitIsometry` and private helpers needed for
  it in `MIPStarRE/QPBT/Algebra/PauliTheorems.lean`.
- Do not edit `Pauli.lean`, `SelfDualBasisTheorems.lean`, blueprint files,
  re-export files, or any other declaration in `PauliTheorems.lean`.
- Do not replace the coordinate isometry with an arbitrary cardinality
  equivalence; compatibility with `kappaVec F` is load-bearing.
- Do not reverse `phi` and `phi.symm`, handle only one Pauli kind, or add a
  positivity or nonemptiness assumption that would exclude `L = 0`.
- Do not add public transport infrastructure, bridge assumptions, conditional
  helpers, axioms, proof bypasses, or new proof holes.
- Use #64 and #66 only after their proofs merge so this target has no
  transitive `sorryAx`. No blueprint proof-status edit belongs to this
  Lean-only packet.

### Acceptance

- The target `sorry` is removed and its public signature and documentation are
  unchanged.
- `#print axioms MIPStarRE.QPBT.exists_qubitIsometry` contains no `sorryAx`.
- After #68, `PauliTheorems.lean` contains no remaining `sorry`, `admit`, new
  axiom, or prohibited bypass.
- The direct file check, proof-debt scan, target build, and full build pass.
- Declaration checking, blueprint rendering/status checks, and
  `git diff --check` pass without a blueprint change.
- The PR r
